array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,11 @@
+/* exported compact */
+
+function compact(array) {
+  const compactArr = [];
+  for (let i = 0; i < array.length; i++) {
+    if (!!(array[i]) === true) {
```

Comment:
In other words, you can just do something like this
`if (array[i])`

Since conditionals will only either evaluate to true or false, we just need to check if the current item is `truthy` or `falsy`

Here's our whole solution
```
function compact(array) {
  const output = [];
  for (let i = 0; i < array.length; i++) {
    const element = array[i];
    if (element) output.push(element);
  }
  return output;
}
```

Reply:
That's more clear and concise. Thank you!